easybuild/easyconfigs/b/beagle-lib/beagle-lib-4.0.1-GCC-12.3.0-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'beagle-lib'
+version = '4.0.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+homepage = 'https://github.com/beagle-dev/beagle-lib'
```

Comment:
@fizwit sorry for nitpicking, but a blank line before `homepage` does make it easier to read, do you mind adding that?

Reply:
@migueldiascosta I went ahead and made that change myself.